docs/source/docs/cli.md

```diff
@@ -219,7 +219,7 @@ configuration can be specified in the configuration file. Please see
   Example usage:
 
   - `--save_png`, for saving the outputs in png format.
-  - `--save_png`, for not saving the outputs in png format.
+  - `--no_png`, for not saving the outputs in png format.
```

Comment:
why has this been changed? what does this have to do with losses?

Reply:
it's a typo and i correct it together ^^

Reply:
You need to add this to the changelog even if it's minor, to keep track, as it doesnt have it's own ticket. It's confusing that unrelated things get changed. We have discussed previously to separate issues into different tickets.

Reply:
done